InventoryConfigurableProduct/Model/IsProductSalableCondition/IsConfigurableProductSalable.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\InventoryConfigurableProduct\Model\IsProductSalableCondition;
+
+use Magento\Catalog\Api\ProductRepositoryInterface;
+use Magento\ConfigurableProduct\Model\Product\Type\Configurable;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\InventorySalesApi\Api\AreProductsSalableInterface;
+use Magento\InventorySalesApi\Api\IsProductSalableInterface;
+
+class IsConfigurableProductSalable implements IsProductSalableInterface
```

Comment:
Doc block is missed I gues...

Reply:
It is not required if the class name describes its purpose well.